modin/engines/base/frame/data.py

```diff
@@ -896,30 +896,37 @@ def _join_index_objects(self, axis, other_index, how, sort):
         """
         Joins a pair of index objects (columns or rows) by a given strategy.
 
+        Unlike Index.join() in Pandas, if axis is 1, the sort is
+        False, and how is "outer", the result will _not_ be sorted.
+
         Parameters
         ----------
-            axis : 0 or 1
-                The axis index object to join (0 - rows, 1 - columns).
-            other_index : Index
-                The other_index to join on.
-            how : {'left', 'right', 'inner', 'outer'}
-                The type of join to join to make.
-            sort : boolean
-                Whether or not to sort the joined index
+
+        axis : 0 or 1 The axis index object to join (0 - rows, 1 - columns).
```

Comment:
I think this was accidentally changed, please revert these documentation formatting changes.

Reply:
Done.